src/node.rs

```diff
@@ -319,11 +327,19 @@ impl<'a> Iterator for Classes<'a> {
     }
 }
 
+/// An iterator over node attributes.
```

Comment:
I'm not actually sure if the iteration *is* over a node's attributes here (because the iterator doesn't have to come from a node's attributes, though I'm not sure from where else it would come :P, but pedantry and I don't know what I'm saying here – uh oh).

```suggestion
/// An iterator over a node's attributes.
```